### ISSUE TITLE
NEO GEO default BIOS automatically when can't find the BIOS it's looking for.

### ISSUE THREAD

**Issue by Cibomatto2002** [libretro/mame2003-plus-libretro#1469]
I have a few NEO GEO games that do not have all the NEO GEO BIOS files so when it tries to load the game it says some files are missing. It would be nice if mame2003-plus would set to the default BIOS automatically when it can't find the BIOS it's looking for.

**Comment by mahoneyt944** [comment]
Ahh yes, there probably should be fallback support when the user selects a bios they don't actually have available.

**Comment by Cibomatto2002** [comment]
I didn't know you could do that in MAME 2003plus I thought they had to be in the same game Zip. Thanks I'm not sure if I should close this or not.

**Comment by mahoneyt944** [comment]
Adding the bios separately(neogeo.zip) will fix your issue, but fallback support is a good idea in general to be more user friendly. The bios files this core uses may differ from current mame. I believe there's another issue open about that. But they are easy to find.

**Comment by mahoneyt944** [comment]
I think fallback support would have to fail at loading the user selection first then try to reload with the default bios. Probably a graceful way of doing this in bios rom loading.

**Comment by mahoneyt944** [comment]
This is what I had in mind https://github.com/libretro/mame2003-plus-libretro/commit/52e8fd474a5ab34087acabfb8610976bcc98b31a

For example if you try to load unibios4 or something but don't actually have it available, we'll try to load the default instead before completely failing.

**Comment by mahoneyt944** [comment]
I had this issue happen to me one time when I had previously selected the unibios4_0 for testing a certain rom. Weeks later I tried loading up another neogeo rom that didn't have this bios in my non-merged set, forgetting I had changed the core option, and I had to manually delete the config to return it to default again. I assume this is the issue the OP wanted to avoid, by falling back to default when the selected bios isn't available.
